content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,155 @@ aliases:
 ---
 
 Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
+The `upgrade` command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 to 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket).
 
-{{% warn %}}
-Back up all data before upgrading with `influx upgrade`.
-{{% /warn %}}
+<!-- TODO summarize effects on other resources -->
+
+## Before you begin: important considerations
```

Comment:
We need another new section --- Secure by Default

If you are currently running InfluxDB and you have:
``` # Determines whether HTTP authentication is enabled.
  # auth-enabled = false```

This is no longer supported in InfluxDB 2.0.  By default authentication is required.
Post upgrade, authentication errors can occur if the username/password credentials within InfluxDB do not match what is currently being used by the various applications, agents, visualization tools, etc.  Consider enabling authentication PRIOR to performing the upgrade to ensure that you have the appropriate credentials in place.